### DISCUSSION TITLE
Veshell

### DISCUSSION THREAD

**Discussion by shanesveller** [material-shell/material-shell#983] [General]
Emailing stargazers privately to recommend your next new project was not a welcome move, please refrain from doing that in the future.

**Comment by PapyElGringo** [comment]
> Emailing stargazers privately to recommend your next new project was not a welcome move, please refrain from doing that in the future.

First sorry if it was an annoyance for you I expected people interested in the project to be interested in the evolution of it.

But don't worry for any following solicitation as it was a single email providing a link to subscribe to a newsletter for those interested in the futures informations
